.github/workflows/ci.yml

```diff
@@ -91,7 +91,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: ['rubocop']
     env:
-      GRAPE_VERSION: '2.0.0'
+      GRAPE_VERSION: '2.2.0'
```

Comment:
I think this is where we expand the matrix to test against multiple versions of Grape, 2.0 and 2.2.

Reply:
What about 2.1, should we test that as well?

Reply:
I've added both 2.1 and 2.2 to CI for now and removed the initial attempt to fix the issues, let's talk about a way to solve this.